samples/word/40-tables/alignment.yaml

```diff
@@ -0,0 +1,138 @@
+order: 3
+id: word-tables-alignment
+name: Get the alignment of a table
+description: 'Get alignment details of a table, a table row, and a table cell.'
+author: OfficeDev
+host: WORD
+api_set:
+    WordApi: '1.3'
+script:
+    content: |
+        $("#get-table-alignment").click(() => tryCatch(getTableAlignment));
+        $("#get-table-row-alignment").click(() => tryCatch(getTableRowAlignment));
+        $("#get-table-cell-alignment").click(() => tryCatch(getTableCellAlignment));
+        $("#setup").click(() => tryCatch(insertTable));
+
+        async function getTableAlignment() {
+          // Get alignment details about the first table in the document.
```

Comment:
What does the alignment of the table mean here? I get row and cell, but is this relative to the Word doc?